doc/dms_setup.md

```diff
@@ -0,0 +1,56 @@
+# Data Migration Service Setup
+
+## Overview
+
+There are a few parts to the setting up of DMS. The account running the DMS terraform needs to have certain AWS service access rights. Configure the VPC peering that is required for the DMS pipelining to other AWS accounts. Set the [create-cloudfoundry] pipeline against the appropriate environment. Make a note of any peering ids that are required. Setup the DMS configuration file named after the environment. Run the `create-cloudfoundry` job called `dms-terraform`.
+
+## Setup Steps
+
+1. Make sure that the concourse AWS account has enough rights to access DMS.
```

Comment:
This can be excluded because we assumed `paas-aws-account-wide-terraform` has been run against an account at the point at which one is working with `paas-cf`

Reply:
👍